### ISSUE TITLE
Donate / archive

### ISSUE THREAD

**Issue by raphink** [camptocamp/puppet-fprint#1]
We no longer use/maintain this module.

@voxpupuli @bastelfreak are you interested in adopting it?

**Comment by bastelfreak** [comment]
I'm not interested in it. and given that it's basically one resource, people might just handle it in their profile.

**Comment by raphink** [comment]
That's true. I hadn't seen it was that basic actually, I thought there was a resource to set the prints up.

I'll just archive it then, sorry for the noise 😁
